parm/fv3lam_post_avblflds.xml

```diff
@@ -5678,6 +5678,41 @@
        <scale>4.0</scale>
     </param>
 
+    <param>
+       <post_avblfldidx>1003</post_avblfldidx>
+       <shortname>ICE_PROB_IFI_FLIGHT_LEVEL</shortname>
+       <pname>ICPRB</pname>
+       <table_info>NCEP</table_info>
+       <fixed_sfc1_type>spec_alt_above_mean_sea_lvl</fixed_sfc1_type>
+       <scale>4.0</scale>
+       <!-- Level units are in 0.1 feet, so final output will be in units of 0.1 meters -->
+       <scale_fact_fixed_sfc1>1</scale_fact_fixed_sfc1>
+       <level>5000. 10000. 15000. 20000. 25000. 30000. 35000. 40000. 45000. 50000. 55000. 60000. 65000. 70000. 75000. 80000. 85000. 90000. 95000. 100000. 105000. 110000. 115000. 120000. 125000. 130000. 135000. 140000. 145000. 150000. 155000. 160000. 165000. 170000. 175000. 180000. 185000. 190000. 195000. 200000. 205000. 210000. 215000. 220000. 225000. 230000. 235000. 240000. 245000. 250000. 255000. 260000. 265000. 270000. 275000. 280000. 285000. 290000. 295000. 300000.</level>
+    </param>
+
+    <param>
+       <post_avblfldidx>1004</post_avblfldidx>
+       <shortname>SLD_IFI_FLIGHT_LEVEL</shortname>
+       <pname>SIPD</pname>
+       <table_info>NCEP</table_info>
+       <fixed_sfc1_type>spec_alt_above_mean_sea_lvl</fixed_sfc1_type>
+       <scale>4.0</scale>
+       <!-- Level units are in 0.1 feet, so final output will be in units of 0.1 meters -->
+       <scale_fact_fixed_sfc1>1</scale_fact_fixed_sfc1>
+       <level>5000. 10000. 15000. 20000. 25000. 30000. 35000. 40000. 45000. 50000. 55000. 60000. 65000. 70000. 75000. 80000. 85000. 90000. 95000. 100000. 105000. 110000. 115000. 120000. 125000. 130000. 135000. 140000. 145000. 150000. 155000. 160000. 165000. 170000. 175000. 180000. 185000. 190000. 195000. 200000. 205000. 210000. 215000. 220000. 225000. 230000. 235000. 240000. 245000. 250000. 255000. 260000. 265000. 270000. 275000. 280000. 285000. 290000. 295000. 300000.</level>
+    </param>
+
+    <param>
+       <post_avblfldidx>1005</post_avblfldidx>
+       <shortname>ICE_SEV_CAT_IFI_FLIGHT_LEVEL</shortname>
+       <pname>ICSEV</pname>
```

Comment:
I would suggest using ICESEV instead of ICSEV. ICESEV has passed WMO approval and ICSEV is just for local use. They mean the same product and are both listed on the grib2 page. https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-19.shtml

Reply:
Okay. I will test this shortly.

Reply:
Actually, no, I can't do that. The existing GRIB files use different numbers for the categories than what WMO wants.

- 0 = none
- 4 = trace
- 1 = light
- 2 = moderate
- 5 = severe

To use the WMO-approved ICESEV, the downstream customers will have to change their code to recognize the 0..4 numbers instead.

I'm going to leave it how it is for now. @HuiyaChuang-NOAA and I will have to bring this up in our next group discussion with AWC, FAA, UCAR, etc.

One option is to generate both products.

Reply:
@SamuelTrahanNOAA You are right. ICESEV is a categorized product. I am going to bring up generating continuous icing severity from UPP too.

Reply:
@SamuelTrahanNOAA Could you incorporate definitions of these new IFI variables in the post_avblflds.xml?

Reply:
It seems I'll need to update the `post_avblfldidx` values since the ones I've been using are already in post_avblflds.xml

Reply:
@SamuelTrahanNOAA The new fields are not defined in post_avblflds.xml yet, ICE_SEV_CAT_IFI_FLIGHT_LEVEL, SLD_IFI_FLIGHT_LEVEL, ICE_PROB_IFI_FLIGHT_LEVEL. I think that's what Wen wanted. @WenMeng-NOAA

Reply:
@SamuelTrahanNOAA That's right, the UPP id 1003 - 1005 are currently used by other variables in post_avblfds.xml, can you reassign these new IFI variables from 1007?

Reply:
Yes. I am already testing that.

Reply:
I did not finish testing this Friday, but I should finish today unless Hera explodes.

Reply:
Although IFI inside the post is working, the results in the GRIB file appear to be junk. I am investigating why now.

Reply:
False alarm!

It turns out the problem was wgrib2 -netcdf which cannot handle the ICESEV. When I disable it in the post, wgrib2 is happy and the results look fine.

I do have a diagnostic output that comes straight out of IFI, and I can tell from that the ICESEV is working. You can output both WMO and NCEP ice severity categories. You just can't feed the WMO categories into the version of wgrib2 I was using (2.1.7).